includes/admin/class-admin.php

```diff
@@ -206,8 +206,12 @@ public function export_form_entries() {
 
                     default:
 
-                        $value              = weforms_get_entry_meta( $entry->id, $column_id, true );
-                        $value              = weforms_get_pain_text( $value );
+                        $value = weforms_get_entry_meta( $entry->id, $column_id, true );
+						$value = weforms_get_pain_text( $value );
+
+						// escape spreadsheet special characters to prevent formula exploits.
+						$value = preg_match( '/^[=+@-].*/', $value ) ? '\'' . $value : $value;

```

Comment:
You can add this to https://github.com/BoldGrid/weforms/blob/9a8f9db1bbc7695be24ab5cc62abc475cd3b6201/includes/functions.php#L1200